x-pack/plugins/security_solution/server/lib/risk_engine/risk_engine_data_client.ts

```diff
@@ -316,12 +304,10 @@ export class RiskEngineDataClient {
             index_patterns: [indexPatterns.alias],
             composed_of: [mappingComponentName],
             template: {
+              lifecycle: {},
```

Comment:
I wasn't sure whether this was all that was needed to opt in to the [lifecycle management for datastreams](https://www.elastic.co/guide/en/elasticsearch/reference/8.11/data-stream-lifecycle.html), but it was [confirmed](https://elastic.slack.com/archives/CE8P96STG/p1696543912113389) by the data management team :+1